packages/channels/botpress-channel-web/src/db.js

```diff
@@ -43,8 +44,19 @@ module.exports = knex => {
         table.timestamp('user_last_seen_on')
         table.timestamp('bot_last_seen_on')
       })
-      .then(function() {
-        return helpers(knex).createTableIfNotExists('web_messages', function(table) {
+      .then(() =>
+        knex.schema
+          .alterTable('web_conversations', table => {
```

Comment:
Does this work on both SQLite and Postgres?

Reply:
Yes it does, SQLite has full support for indices

Reply:
Awesome!